extensions/amp-lightbox-gallery/1.0/README.md

```diff
@@ -0,0 +1,274 @@
+its description, showing "picture of cn tower".
+
+```html
+<img
+  id="hero-img"
+  lightbox="toronto"
+  src="https://picsum.photos/1600/900?image=1075"
+  layout="responsive"
+  width="1600"
+  height="900"
+  alt="picture of cn tower"
+>
+```
+
+#### Implement Thumbnail Previews
+
+Lightboxed items have a thumbnail gallery view. you can optionally specify a
+thumbnail item for your lightboxed element via the attribute
+`lightbox-thumbnail-id` that references the `id` of an `<img>` element with
+`layout="nodisplay"`.
+
+```html
+<bento-youtube
+  width="480"
+  height="270"
+  layout="responsive"
+  data-videoid="lbtcb7yls8y"
+  lightbox-thumbnail-id="my-thumbnail-img"
+>
+</bento-youtube>
+
+<img
+  id="my-thumbnail-img"
+  width="200"
+  height="200"
+  layout="nodisplay"
+  src="https://picsum.photos/200/200?image=1074"
+>
+```
+
+If no thumbnail is specified, `<img>` elements will be cropped per
+`object-fit: cover`, `<bento-video>` will use the image `src` specified in its
+`poster` attribute, and placeholder images will be used for lightboxed elements
+that have one.
+
+### Actions
+
+### `open`
+
+Opens the lightbox gallery. can be triggered by tapping another element, if you
+specify the image `id`: `on="tap:amp-lightbox-gallery.open(id='image-id')"`.
+
+### Layout And Style
+
+Each bento component has a small css library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-lightbox-gallery-1.0.css">
+```
+
+## Preact/React Component
+
+The preact/react version of the bentolightboxgallery functions differently than the web component version. The following example will demonstrate the use of `<BentoLightboxGallery>` as a functional component.
+
+### Example: Import Via npm
+
+[example preview="top-frame" playground="false"]
+
+```sh
+npm install @ampproject/bento-lightbox-gallery
+```
+
+```javascript
+import react from 'react';
+import {
+  bentolightboxgalleryprovider,,
+  withbentolightboxgallery,
+} from '@ampproject/bento-lightbox-gallery/react';
+
+function app() {
+  return (
+      <bentolightboxgalleryprovider>
+        <withbentolightboxgallery>
+          <img src="https://images.unsplash.com/photo-1562907550-096d3bf9b25c"/>
+        </withbentolightboxgallery>
+      </bentolightboxgalleryprovider>
+  );
+}
+
+```
+
+[/example]
+
+#### Example Using BentoBaseCarousel
+
+`<BentoLightboxGallery>` can be used with a `<BentoBaseCarousel>` child in order to lightbox all of the carousel's children. As you navigate throught the carousel items in the lightbox, the original carousel slides are synchronised so that when the lightbox is closed, the user ends up on the same slide as they were originally on. Currently, only the `type='slides'` carousel is supported.
```

Comment:
`type` does not exist on `BentoBaseCarousel`
```suggestion
`<BentoLightboxGallery>` can be used with a `<BentoBaseCarousel>` child in order to lightbox all of the carousel's children. As you navigate throught the carousel items in the lightbox, the original carousel slides are synchronised so that when the lightbox is closed, the user ends up on the same slide as they were originally on. 
```